src/App.tsx

```diff
@@ -5,19 +5,28 @@ import React from 'react';
 import { DEFAULT_NETWORK } from 'consts';
 import { Providers } from 'Providers';
 import { ThemesProvider } from 'contexts/Themes';
+import { i18next } from 'locale';
+import { I18nextProvider } from 'react-i18next';
 
 const App: React.FC = () => {
   let network = localStorage.getItem('network');
+  let locale = localStorage.getItem('locale');
 
   if (network === null) {
     network = DEFAULT_NETWORK;
     localStorage.setItem('network', network);
   }
+  if (locale === null) {
+    locale = 'en';
```

Comment:
can we change this hard-coded value to DEFAULT_LOCALE and use it in the initial setup?